src/analysis/gMCS/calculateMCS.m

```diff
@@ -347,48 +415,125 @@
     ctype(var.delw) = 'C';
     ctype(var.x) = 'C';
 
-% Introduce all data in a structure
-    MILPproblem.A = A;
-    MILPproblem.b = rhs;
-    MILPproblem.c = obj;
-    MILPproblem.lb = lb;
-    MILPproblem.ub = ub;
-    MILPproblem.csense = csense;
-    MILPproblem.vartype = ctype;
-    MILPproblem.osense = 1;
-    MILPproblem.x0 = [];
+% Cplex - sense of the optimization
+    sense = 'minimize';
+
+% Cplex - Introduce all data in a Cplex structure
```

Comment:
Same comment here - actually, you may also be able to write a separate internal function to shorten and simplify the code.